cryodrgn/types.py

```diff
@@ -6,13 +6,13 @@
 """
 from typing import TYPE_CHECKING, Union, List
 import numpy as np
+import torch
 
 
 if TYPE_CHECKING:  # Set if type-checking
     # Avoid importing any cryodrgn-specific submodules outside this block.
-    from cryodrgn.mrc import LazyImage
 
-    ImageArray = Union[np.ndarray, List[LazyImage]]
+    ImageArray = Union[np.ndarray, torch.Tensor]
```

Comment:
Would it be possible to narrow this further and always use `Tensor` or always use `ndarray`?